services/smartcards/writePollWorkerCard.py

```diff
@@ -12,13 +12,24 @@
 election_bytes = f.read()
 f.close()
 
+long_value = None
+
+if len(sys.argv) > 2:
```

Comment:
Can we either add a comment explaining what the expected args are and what they do, or handle `--help`/`-h` args to print expected usage?

Reply:
Comment!